vyper/ast/parse.py

```diff
@@ -144,9 +146,11 @@ def annotate_python_ast(
         The annotated and optimized AST.
     """
 
-    tokens = asttokens.ASTTokens(source_code, tree=cast(Optional[python_ast.Module], parsed_ast))
+    tokens = asttokens.ASTTokens(
+        original_source, tree=cast(Optional[python_ast.Module], parsed_ast)
```

Comment:
```suggestion
        python_source, tree=cast(Optional[python_ast.Module], parsed_ast)
```
I think this might work.